src/KW_button.h

```diff
@@ -28,46 +28,51 @@
 
 /**
  * \file KW_button.h
- * 
+ *
  * Declares functions for creating and managing buttons
  */
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-
 /**
  * \brief   Creates a button widget.
- * \details The button widget is a composite widget. It uses the second 3x3 tile block (next to the frame block) to render its lines.
- * \param   gui The KW_GUI instance that will hold this button.
- * \param   parent The parent widget of this button.
- * \param   label The label widget to print inside the button.
+ * \details The button widget is a composite widget. It uses the second 3x3 tile
+ * block (next to the frame block) to render its lines. \param   gui The KW_GUI
```

Comment:
This looks broken, each `\param` should be on the same line

Reply:
that looks like a relatively easy fix with `CommentPragmas:  '^\\.+'`, so I'll add and try that, i just need to commit & PR my radiobutton code